computation-local/src/main/java/eu/itesla_project/computation/local/LocalExecutor.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.computation.local;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public interface LocalExecutor {
+
+    int execute(String program, List<String> args, Path out, Path workingDir, Map<String, String> env) throws IOException, InterruptedException;
```

Comment:
It could be a good idea to have an "err" parameter to distinguish out and err outputs.